x-pack/plugins/security_solution/server/endpoint/routes/metadata/index.ts

```diff
@@ -113,6 +122,12 @@ export function registerEndpointRoutes(router: IRouter, endpointAppContext: Endp
         return res.notFound({ body: 'Endpoint Not Found' });
```

Comment:
I have not been picking up on this, but I think all returned static error messages should be i18n'ed 
(maybe we have discussed this already?)

Reply:
> I have not been picking up on this, but I think all returned static error messages should be i18n'ed
(maybe we have discussed this already?)

I do not think this applies to server side code, usually the approach is error code. This is the pattern system wide. But we will certainly review that approach.